Doc: This patch new guides with titles mentioned below

koolhead17 opened the pull request:
- Minio- configuration file
- Running multiple Minio server on single instance
- Setup replication between two sites running Minio.

koolhead17 committed:
Doc: Added new documents related to

koolhead17 committed:
Doc: Modified the markdown file names to match with title.

koolhead17 committed:
Doc: Modified the configuration files guide to add recent version
config.json

brendanashworth commented on docs/minio-server-configuration-files-guide.md:
```diff
@@ -0,0 +1,107 @@
+
+``version``  talks about the version of the file.
+
+``credential`` stores authenctication credentials for your Minio server. If you want to provide your own custom access/secret key you will have to modify it and run Minio.
```
authenctication -> authentication

brendanashworth commented on docs/setup-replication-between-two-sites-running-minio.md:
````diff
@@ -0,0 +1,120 @@
+
+```
+
+## 4. Settup crontab
````
Settup -> Setup

brendanashworth commented on docs/setup-replication-between-two-sites-running-minio.md:
````diff
@@ -0,0 +1,120 @@
+```
+You can run `` mc diff --help`` on your terminal for more details.
+
+Note: We are going to inrtoduce continuous replication feature in `mc mirror` which will enable the sites to be in sync without having the need to set up cron job.
````
inrtoduce -> introduce

koolhead17 committed:
Doc: Fixed typos/spelling mistakes from the docs.

harshavardhana commented on docs/minio-server-configuration-files-guide.md:
```diff
@@ -0,0 +1,107 @@
+
+``region``: We are following S3 specs and hence the region.
+
+``logger``: We have introduced new notification feature in Minio, stay tuned will talk about this in saperate post.
```
This is not correct explanation, 'notify' is the new field. 'logger' is an older one for error logs.

koolhead17 replied:
Fixing it right away.

harshavardhana commented on docs/setup-replication-between-two-sites-running-minio.md:
```diff
@@ -0,0 +1,120 @@
+datadir="/home/minio/minio1-data/"
+minio2="minio2/mbucket"
+MC_PATH="/home/minio/mc"
+$MC_PATH --quiet  mirror --force $minio1 $minio2
```
We have to talk about '--force' here since it would overwrite the destination files.

koolhead17 replied:
already mentioned above.

harshavardhana commented on docs/setup-replication-between-two-sites-running-minio.md:
````diff
@@ -0,0 +1,120 @@
+```
+You can run `` mc diff --help`` on your terminal for more details.
+
+Note: We are going to introduce continuous replication feature in `mc mirror` which will enable the sites to be in sync without having the need to set up cron job.
````
'setup a cron job'  - 'set up' is a single word.

harshavardhana commented on docs/setup-replication-between-two-sites-running-minio.md:
````diff
@@ -0,0 +1,120 @@
+*/30 * * * * /home/minio/minio.sh 
+```
+
+## mc diff
````
There is no need to talk about 'mc diff' i think - because 'mc diff' does bi-directional 'diff' , which is different than how 'mc mirror' behaves.

harshavardhana commented on docs/setup-replication-between-two-sites-running-minio.md:
````diff
@@ -0,0 +1,120 @@
+
+```
+
+Setup executable permissions on script so that cron can execute in every 30 minutes.
````
'Set executable permissions on the script before adding a cron entry.'

harshavardhana commented on docs/setup-replication-between-two-sites-running-minio.md:
````diff
@@ -0,0 +1,120 @@
+$ chmod 755 /home/minio/minio.sh
+```
+```sh
+
````
'Set a new cron entry to run `minio.sh` script once every 30mins`

harshavardhana commented on docs/setup-replication-between-two-sites-running-minio.md:
```diff
@@ -0,0 +1,120 @@
+### Script
+Add crontab configuration on `minio1` providing path of data directory, ``minio1-data``. 
+
+ ``--force`` option with ``mc mirror`` is used to keep the contents latest in destination.  
```
`--force` option overwrites the destination contents, this would keep your contents in sync.

koolhead17 committed:
Doc: This patch will improve the content of
minio server configuration files guide and setup replication between two
sites running minio docs.

harshavardhana commented on docs/setup-replication-between-two-sites-running-minio.md:
```diff
@@ -79,9 +79,11 @@ Cron is a Unix/Linux system utility by which you can schedule a task process for
 Add crontab configuration on `minio1` providing path of data directory, ``minio1-data``. 
 
- ``--force`` option with ``mc mirror`` is used to keep the contents latest in destination.  
+``--force`` option with ``mc mirror``  overwrites the destination contents, this would keep your contents in sync..  
```
remove the '..' at the end.

koolhead17 replied:
fixed

harshavardhana commented on docs/setup-replication-between-two-sites-running-minio.md:
````diff
@@ -92,28 +94,22 @@ $MC_PATH --quiet  mirror --force $minio1 $minio2
 $ chmod 755 /home/minio/minio.sh
 ```
+
+Set a new cron entry to run ``minio.sh`` script once every 30mins
````
missing a '.' at the end.

koolhead17 replied:
fixed

koolhead17 committed:
Doc: Fixed typos from setup replication between two sites running minio
doc.
